rpc/query_server.go

```diff
@@ -782,8 +782,8 @@ func (s *QueryServer) EthGetTransactionReceipt(hash eth.Data) (*eth.JsonTxReceip
 	txResults, err := s.BlockStore.GetTxResult(blockResult.Block.Data.Txs[txReceipt.TransactionIndex].Hash())
 	if err != nil {
 		if strings.Contains(errors.Cause(err).Error(), "not found") {
-			// return nil response if cannot find hash
-			return nil, nil
+			// return receipts without blockresult info
```

Comment:
If `blockStore.GetTxResult` doesn't have the data try `blockStore.GetBlockResults`.

Reply:
Done